### ISSUE TITLE
_ and - cause different scopes to be chosen

### ISSUE THREAD

**Issue by alexr00** [jeff-hykin/better-shell-syntax#37]
## The code with a problem is:
```sh
alias brew_list="brew leaves"
alias brew-list="brew leaves"

```

## It looks like:
![image](https://user-images.githubusercontent.com/38270282/216955577-4de174e0-b428-4e29-9270-8ac52f328ff7.png)

## It should look like:
On the first line, `brew_list` gets tokenized as `variable.other.assignment.shell`. On the second line, `brew-list` gets tokenized as `string.unquoted.argument.shell`. I would expect to both have the same scope.

Originally from: https://github.com/microsoft/vscode/issues/173224

**Comment by jeff-hykin** [comment]
easy fix, published with `v1.3.2`

**Comment by alexr00** [comment]
Thanks!
